subscription-engine/kafka-ws/index.js

```diff
@@ -46,7 +46,12 @@ server.listen(serverCfg.port, serverCfg.host, async () => {
   logger.info(server.address());
 
   // Initializes the sticky tarball
-  await websocketTarball.init();
+  try {
```

Comment:
Maybe you can init the try at line 25

Reply:
I didn't understand where to try catch. It would be in this same script?

Reply:
```js
try{ 
const logger = new Logger();
let server = null;
if (serverCfg.tls) {
  logger.info('Initializing the HTTP server (Using TLS Protocol)...');
  const options = {
    cert: fs.readFileSync(serverCfg.tls_cert_file),
    key: fs.readFileSync(serverCfg.tls_key_file),
    ca: [fs.readFileSync(serverCfg.tls_ca_file)],
    rejectUnauthorized: true,
    requestCert: serverCfg.request_cert,
  };
  server = https.createServer(options, application.expressApp);
} else {
  logger.info('Initializing the HTTP server...');
  server = http.createServer(application.expressApp);
}
/* Configures the application's HTTP and WS routes */
application.configure(server);
server.listen(serverCfg.port, serverCfg.host, async () => {
  logger.info('HTTP server is ready to accept connections!');
  logger.info(server.address());
 await websocketTarball.init()
  } catch (err) {....
```

Reply:
It needs to be inside the handler `server.listen()`